api/baseline-api-v1.0.0.yaml

```diff
@@ -0,0 +1,4382 @@
+          type: string
+        api_endpoint:
+          type: string
+        messaging_endpoint:
+          type: string  
+    ProtocolMessage:
+      title: Protocol Message
+      type: object
+      description: baseline protocol message
+      properties:
+        baseline_id:
+          type: string
+          format: uuid
+          readOnly: true
+        opcode:
+          type: string
+        sender:
+          type: string
+        recipient:
+          type: string
+        shield:
+          type: string
+        identifer:
+          type: string
+          format: uuid
+          readOnly: true
+        signature:
+          type: string
+        type:
+          type: string
+        payload:
+          $ref: '#/components/schemas/ProtocolMessagePayload'
+    ProtocolMessagePayload:
+      title: Protocol Message Payload
+      description: Baseline protocol message payload
+      properties:
+        object:
+          type: object
+        proof:
+          type: string
+        type:
+          type: string
+        witness:
+          type: object
+    ProveRequest:
+      title: Prove Request
+      required:
+        - curve
+        - identifier
+        - name
+        - provider
+        - proving_scheme
+      type: object
+      properties:
+        identifier:
+          type: string
+        proving_scheme:
+          type: string
+        curve:
+          type: string
+        provider:
+          type: string
+        name:
+          type: string
+      example:
+        identifier: cubic
+        proving_scheme: groth16
+        curve: BN256
+        provider: gnark
+        name: my 1337 circuit
+    Secret:
+      title: Secret
+      required:
+        - description
+        - name
+        - type
+        - vault_id
+        - value
+      type: object
+      properties:
+        id:
+          type: string
+          format: uuid
+          readOnly: true
+        created_at:
+          type: string
+          readOnly: true
+        vault_id:
+          type: string
+          readOnly: false
+        type:
+          type: string
+          readOnly: false
+        name:
+          type: string
+          readOnly: false
+        description:
+          type: string
+          readOnly: false
+        value:
+          type: string
+          readOnly: false
+      example:
+        id: e9138dd7-b429-410a-bfa8-2a506d9e4297
+        created_at: '2021-04-17T20:35:34.062+0000'
+        vault_id: 22e48752-7bff-4daa-89b3-91053bd509f2
+        type: sample secret
+        name: 'secret #1'
+        description: this secret is being stored for demonstration purposes
+    Security:
+      title: Security
+      required:
+        - egress
+        - ingress
+      type: object
+      properties:
+        egress:
+          type: string
+        ingress: {}
+      example:
+        egress: '*'
+        ingress:
+          0.0.0.0/0:
+            tcp:
+              - 4222
+            udp: []
+    Token:
```

Comment:
part of another standard. only reference other standards when they exist and do not specify here